packages/react/src/runtimes/edge/streams/toolResultStream.ts

```
@@ -1,39 +1,89 @@
-            );
-          });
-      }
+  if (tool.parameters instanceof z.ZodType) {
+    const result = tool.parameters.safeParse(toolCall.args);
+    if (!result.success) {
+      executeFn =
+        tool.experimental_onSchemaValidationError ??
+        (() => {
+          throw (
+            "Function parameter validation failed. " +
+            JSON.stringify(result.error.issues)
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)